source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py

```diff
@@ -988,7 +988,12 @@ def transform_points(
 
 @torch.jit.script
 def unproject_depth(depth: torch.Tensor, intrinsics: torch.Tensor) -> torch.Tensor:
-    r"""Unproject depth image into a pointcloud.
+    r"""Unproject depth image into a pointcloud. This method assumes that depth
+    is provided orthagonally relative to the image plane, as opposed to absolutely relative to the camera's
+    principal point (perspective depth). To unproject a perspective depth image, use
+    :meth:`convert_perspective_depth_to_orthogonal_depth` to convert
+    to an orthogonal depth image prior to calling this method as otherwise the
+    created point cloud will be distorted, especially around the edges.
```

Comment:
```suggestion
    to an orthogonal depth image prior to calling this method. Otherwise, the
    created point cloud will be distorted, especially around the edges.
```